lib/components/app/app-menu.tsx

```diff
@@ -176,85 +138,80 @@ class AppMenu extends Component<
 
     const { isPaneOpen } = this.state
     const { SvgIcon } = this.context
+    const buttonLabel = isPaneOpen
+      ? intl.formatMessage({ id: 'components.AppMenu.closeMenu' })
+      : intl.formatMessage({ id: 'components.AppMenu.openMenu' })
+
     return (
       <>
-        <div
-          aria-label={
-            isPaneOpen
-              ? intl.formatMessage({ id: 'components.AppMenu.closeMenu' })
-              : intl.formatMessage({ id: 'components.AppMenu.openMenu' })
-          }
-          className="app-menu-icon"
+        <button
+          aria-controls="app-menu"
+          aria-expanded={isPaneOpen}
+          aria-label={buttonLabel}
+          className={`app-menu-icon ${isPaneOpen ? 'open' : ''}`}
           onClick={this._togglePane}
-          onKeyDown={this._togglePane}
-          role="button"
-          tabIndex={0}
         >
-          <span className={isPaneOpen ? 'menu-left-x' : 'menu-top-line'} />
-          <span className={isPaneOpen ? '' : 'menu-middle-line'} />
-          <span className={isPaneOpen ? 'menu-right-x' : 'menu-bottom-line'} />
-        </div>
+          <span />
+          <span />
+          <span />
+        </button>
+        {/* Use a button for skipping navigation. A regular <a> element would modify the URL,
```

Comment:
Is there no way to use `preventDefault` instead? That feels less bad than using a button...

Reply:
The anchor + href implementation is more complicated unfortunately, the main complicated things being generating the URL (e.g. `#content/?ui_activeSearch=...`) and removing the `#content` part of the URL once the link is clicked.